extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/JsonSpecFileGenerator.scala

```diff
@@ -35,6 +35,7 @@ object JsonSpecFileGenerator {
     writeCommandSpecJson("table", TableCommands.data ++ IcebergCommands.data)
     writeCommandSpecJson("function", FunctionCommands.data)
     writeCommandSpecJson("scan", Scans.data)
+    writeCommandSpecJson("function_resource", Functions.data)
```

Comment:
"function_resource" for a type of resource name could be confused, especially with "function".  Any better suggestion for this?

Reply:
> "function_resource" for a type of resource name could be confused, especially with "function". Any better suggestion for this?

To obtain, we can divide resources into two types, command and entity, and generate two kinds of json files like function_command and function_entity

Reply:
It's quite confusing, `gen.Functions` generates `function_resource_command_spec.json`, while `gen. FunctionCommands` generates `function_command_spec.json`. Shall we fix it in place to allow each generator defines their file name for spec generation instead of a unified `_command_spec` suffix? @yaooqinn

Reply:
I agree with it, allow each generator defines their file name for spec generation.

Reply:
Yeah, I think we could make it this way.

resource| spec json file | spec json file (suggested) |
--- | --- | --- |
Database | `database_command_spec.json` | `database_command_spec.json` |
Table | `database_command_spec.json` | `database_command_spec.json` |
Function | `function_command_spec.json` | `function_command_spec.json` |
Scan （Query+Function Scan） | `scan_command_spec.json` | [Changed] `query_scan_spec.json`

Reply:
Let's skip this suggestion, and continue to use existed spect files.